Specta/Specta/SPTSpec.m

```diff
@@ -145,6 +145,7 @@ - (void)spt_runExample:(SPTCompiledExample *)example {
     example.block(self);
   } else if (!example.pending) {
     self.spt_skipped = YES;
+    NSLog(@"Skipping Test");
```

Comment:
I believe the `example` has a property for its name--why not log that as well, so users know which tests are being skipped?